### PR TITLE
Documentation changes

### PR DESCRIPTION
**Pull request by prabhuteja12**
Adds CDDB, Core50, L2P documentation on the benchmarks listing page. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: doc/benchmarking/renate_benchmarks.rst [hunk 1]
```diff
@@ -173,6 +180,14 @@ The following table contains the list of supported datasets.
       - multiple
       - Any `Hugging Face dataset <https://huggingface.co/datasets>`__ can be used. Just prepend the prefix ``hfd-``, e.g., ``hfd-rotten_tomatoes``. Select input and target columns via ``config_space``, e.g., add ``"input_column": "text", "target_column": "label"`` for the `rotten_tomatoes <https://huggingface.co/datasets/rotten_tomatoes>`__ example.
       - Please refer to `the official documentation <https://huggingface.co/datasets>`__.
+    * - CDDB
+      - Image Classification
+      - 2 classes from 5 domains of generated images using various techniques. Numbers vary across domains.
```
Comment (wistuba) [inline doc/benchmarking/renate_benchmarks.rst:185]: can we make domains explicit?

"2 classes from 5 domains of generated images using various techniques" this description wouldn't be clear to me. how about this:
- Image Classification: deepfake detection
- 2 classes, 5 domains, each representing an image generation technique: ...

File: doc/benchmarking/renate_benchmarks.rst [hunk 2]
```diff
@@ -173,6 +180,14 @@ The following table contains the list of supported datasets.
...
+      - Li, Chuqiao, et al. A continual deepfake detection benchmark: Dataset, methods, and essentials. IEEE/CVF Winter Conference on Applications of Computer Vision. 2023.
+    * - CORe50
+      - Object Recognition - image classfication
```
Comment (wistuba) [inline doc/benchmarking/renate_benchmarks.rst:188]: Can we stick to the format used earlier, i.e., Image Classification?

File: doc/benchmarking/renate_benchmarks.rst [hunk 3]
```diff
@@ -173,6 +180,14 @@ The following table contains the list of supported datasets.
       - multiple
       - Any `Hugging Face dataset <https://huggingface.co/datasets>`__ can be used. Just prepend the prefix ``hfd-``, e.g., ``hfd-rotten_tomatoes``. Select input and target columns via ``config_space``, e.g., add ``"input_column": "text", "target_column": "label"`` for the `rotten_tomatoes <https://huggingface.co/datasets/rotten_tomatoes>`__ example.
       - Please refer to `the official documentation <https://huggingface.co/datasets>`__.
+    * - CDDB
+      - Image Classification
+      - 2 classes from 5 domains of generated images using various techniques. Numbers vary across domains.
+      - Li, Chuqiao, et al. A continual deepfake detection benchmark: Dataset, methods, and essentials. IEEE/CVF Winter Conference on Applications of Computer Vision. 2023.
+    * - CORe50
```
Comment (wistuba) [inline doc/benchmarking/renate_benchmarks.rst:187]: This refers to the name we use to load the data as part of benchmarking. Right now, this should be `Core50`.

File: doc/benchmarking/renate_benchmarks.rst [hunk 4]
```diff
@@ -173,6 +180,14 @@ The following table contains the list of supported datasets.
...
+      - Object Recognition - image classfication
+      - 10 classes of 7 training domains, 3 other domains used for evaluation
```
Comment (wistuba) [inline doc/benchmarking/renate_benchmarks.rst:189]: There are 8 domains. The 3 other domains is irrelevant for evaluation. Rather mention that there is only 1 test dataset which will be used as test set for each task. Same comment as before "10 classes of 7 training domains". What does that mean? Rather something like "50 classes, 8 domains (0-7)." Note, there are 50 classes, not 10.

**Review by wistuba — CHANGES_REQUESTED** [review]

File: doc/benchmarking/renate_benchmarks.rst (hunk 3 above)
Comment (prabhuteja12) [reply doc/benchmarking/renate_benchmarks.rst:187]: ok

File: doc/benchmarking/renate_benchmarks.rst (hunk 1 above)
Comment (prabhuteja12) [reply doc/benchmarking/renate_benchmarks.rst:185]: ok

File: doc/benchmarking/renate_benchmarks.rst (hunk 2 above)
Comment (prabhuteja12) [reply doc/benchmarking/renate_benchmarks.rst:188]: ok

**Review by wistuba — APPROVED** [review]

**Pull request merged**